cypress/integration/cbl/admin/tasks-manager.js

```diff
@@ -178,6 +178,39 @@ describe('CBL / Admin / Tasks Manager', () => {
         // wait for data load
         cy.wait('@tasksData');
 
+        // verify create button not disabled
+        cy.extGet('slate-tasks-manager-appheader button[action=create]')
+            .should('exist')
+            .should('not.have.class', 'x-btn-disabled')
+
+        // verify edit button disabled
+        cy.extGet('slate-tasks-manager-appheader button[action=edit]')
+            .should('exist')
+            .should('have.class', 'x-btn-disabled')
+
+        // verify delete button disabled
+        cy.extGet('slate-tasks-manager-appheader button[action=delete]')
+            .should('exist')
+            .should('have.class', 'x-btn-disabled')
+
+        // select the first grid cell
+        cy.root()
+            .get('.x-grid-cell-inner')
```

Comment:
In this case, using `cy.root()` is redundant:

```suggestion
        cy.get('.x-grid-cell-inner')
```

Reply:
resolved [3a98c74](https://github.com/SlateFoundation/slate-cbl/pull/732/commits/3a98c74dd03dbe6899a345a07d1ee26c56d779a9)